docs/madr/decisions/038-transition-to-new-policies.md

```diff
@@ -0,0 +1,473 @@
+# Transition to new policies
+
+- Status: accepted
+
+## Context and Problem Statement
+
+The current state of the policies as of Kuma 2.6.x:
+* new policies are in GA, they're a recommended way to configure new deployments in Kuma
+* old policies are not deprecated, they're still in use by majority of Kuma users
+
+The goal is to provide a way to transition from old policies to new policies. Having a smooth transition is going to enable 
+us to deprecate old policies in the future.
```

Comment:
We should refresh people's mind in this MADR by mentioning that old policies do not apply as soon as new policies are applied. I think it's important to understand this MADR

Reply:
added a note under the `current state` section